libs/vkd3d/state.c

```diff
@@ -4562,6 +4566,13 @@ static HRESULT d3d12_pipeline_state_init_graphics_create_info(struct d3d12_pipel
         goto fail;
     }
 
+    /* Tests show that D3D12 drivers behave as if D3D12_PIPELINE_STATE_FLAG_DYNAMIC_DEPTH_BIAS
+     * was always set, howevert doing that would invalidate existing pipeline caches, so avoid
+     * this until proven necessary. */
+    if ((desc->flags & D3D12_PIPELINE_STATE_FLAG_DYNAMIC_DEPTH_BIAS) &&
+            (graphics->ds_desc.depthTestEnable))
```

Comment:
Depth bias affects FragCoord.z I think, which exists without depth test I think?

Reply:
As per discord discussion, this will need some additional testing, the way we currently deal with DSV with no depth test might be incorrect.